frontend/src/components/BrushColorTile.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div :style="style" v-on:click="onTileClicked"></div>
+</template>
+
+<script>
+export default {
+  name: "BrushColorTile",
+  props: {
+    color: String
+  },
+  computed: {
+    style() {
+      return {
+        height: "20px",
+        width: "20px",
+        borderRadius: "30%",
+        backgroundColor: this.color
+      };
```

Comment:
i'm ok with this for this PR, but @gordon-cheung , I remember you mentioned something about using scss before. In Angular, usually there's a "component" directory for every component.. example..

ComponentA/
 - ComponentA.html
 - ComponentA.scss
 - ComponentA.ts (typescript)
 - ComponentA.spec (test file)

It doesn't look like this is the case with Vue, but do you know if we have scss files for every vue file? or is the scss inside the .vue?

Reply:
I think you can be done both ways. Not sure what is better though.